packages/core/lib/process-request.ts

```diff
@@ -97,6 +97,7 @@ export const processRequest = async <TContext = {}, TRootValue = {}>(
     validate = defaultValidate,
     validationRules,
     variables,
+    allowedSubscriptionHttpMethods = ["GET"],
```

Comment:
```suggestion
    allowedSubscriptionHttpMethods = ["GET", "POST"],
```

Reply:
We should also allow POST be default even though sse is usually not done via POST.

Reply:
@dan-lee After some more discussions I think we should allow POST by default, can you please do this change and then rebase?

Reply:
@n1ru4l Alright, I applied the change 👍